src/fr/composants/message-derreur/code.md

```diff
@@ -6,4 +6,41 @@ tags: ['errormessageFR', 'code']
 date: "git Last Modified"
 ---
 
-## Code
\ No newline at end of file
+## Création d'un message d'erreur
+
+Rédigez un message d'erreur pour chaque contrainte de réponse d'un composant, lorsque ce composant est requis.
+
+Tous les composants du système de conception GC sont déjà livrés avec une gestion des erreurs par défaut pour les composants de formulaire requis.
+
+## Codage et accessibilité des messages d'erreur
+
+### Rédigez des messages d'erreur pour faciliter l'accomplissement des tâches.
+
+- Assurez-vous que l'on puisse accéder immédiatement aux contrôles du formulaire associés au libellé auquel le message d'erreur est attaché.
+- Rédigez un message d'erreur spécifique pour éviter qu'il s'affiche à nouveau alors que l'utilisateur·rice pensait l'avoir corrigé.  
+- Ne proposez pas d'étapes supplémentaires. Précisez ce qu'il faut faire pour corriger l'erreur pour que l'on puisse prendre des mesures et soumettre de nouveau — voire revalider — les modifications sans devoir suivre des étapes supplémentaires.
+- Fournissez une ou deux solutions pour corriger l'erreur, plutôt qu'une démarche en plusieurs étapes.
+
+### Rédigez un message d'erreur classique
+
```

Comment:
There might be a sentence missing here in the french Figma file - the English content has a sentence between the heading and the bullet list.

Reply:
"Provide simple, fast guidance that's in context and interrupts the flow of action as minimally as possible:"